.k8s/deploy-k8s.groovy

```diff
@@ -0,0 +1,67 @@
+//Groovy scripts throw lots of WARNING messages, but it is fine
+//https://issues.apache.org/jira/browse/GROOVY-8339
+
+
+def DOCKER_REG = System.getenv("DOCKER_REGISTRY")
+if (!DOCKER_REG) {
+    println "DOCKER_REGISTRY environment variable not found!"
+    System.exit(1)
+}
+
+//The name of the preface for the docker images
+def DOCKER_IMAGE_NAME = "cnxta/ion-search"
+def DOCKER_SOLR_IMAGE = "cnxta/search-solr"
+
+//The path to the Docker Wrapper
+//IMPORTANT!
+//>>> The base docker script has to be in the same location as this file
+
```

Comment:
❓ If we don't need the docker wrapper, should we remove references to it in these scripts?

Reply:
Yeah, I'll remove that in the next commit